packages/uncompyle6.vm/uncompyle6.vm.nuspec

```
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>uncompyle6.vm</id>
+    <version>3.9.2</version>
+    <authors>rocky</authors>
+    <description>A cross-version Python bytecode decompiler</description>
```

[nitpick] uncompyle6 doesn't work on the latest Python versions. I think the latest supported is 3.8. I think we should add this:
```suggestion
    <description>A decompiler for Python 1.0-3.8.</description>
```

@sara-rn you can edit the issue and then re-trigger the automation **without creating a new issue**. Let me know if this is not clear.